README.md

```diff
@@ -835,6 +835,12 @@ support preserving stats between role reload
 (see [tarantool/metrics#334](https://github.com/tarantool/metrics/issues/334)),
 thus this feature will be unsupported for `metrics` driver.
 
+### VERSION
+
+`VERSION` - is a crud module version constant.
+
+Use `crud.VERSION` on "routers" to get module version, or `_crud.VERSION` - on storages.
```

Comment:
Why we use globals and different API for routers and storages? There are no supported cases for using `_crud` now, all interaction are processed with `require('crud').something`

Reply:
> There are no supported cases for using _crud: And what is this: https://github.com/tarantool/crud/blob/e709a87918a91a35181fff91fed4ce246bd57620/crud.lua#L123 ?
and this;
https://github.com/tarantool/crud/blob/e709a87918a91a35181fff91fed4ce246bd57620/crud/select.lua#L103
and so on...?

Reply:
It's internal and should be used only by crud itself, not by user